### PR TITLE
Feature/dapp 546

### PR DESCRIPTION
**Pull request by NickNish09**
# Description

- Add support cause page components (`/promoters/support-cause`)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Test A

### CODE REVIEW

**Review by thaisribon — COMMENTED** [review]
I think the font-size of "Donate with a community" and of the causes buttons are not matching with figma

**Review by thaisribon — APPROVED** [review]

**Review by thaisribon — COMMENTED** [review]
the currency dropdown has the wrong font-weight, i just saw it :/

File: src/components/atomics/Dropdown/styles.ts [hunk 1]
```diff
@@ -34,6 +28,7 @@ export const Input = styled.div`
     border: none;
     border-radius: 4px;
     box-sizing: border-box;
+    font-weight: 600;
```
Comment (thaisribon) [inline src/components/atomics/Dropdown/styles.ts:31]: I think you can use ${defaultComponentTextMedium}

**Review by thaisribon — APPROVED** [review]

File: src/components/atomics/Dropdown/styles.ts (hunk 1 above)
Comment (NickNish09) [reply src/components/atomics/Dropdown/styles.ts]: updated

**Pull request merged**